### PR TITLE
[v0.37] fix / add market validators

### PR DESCRIPTION
**Pull request by vic-en**
Fixes https://github.com/CoinAlpha/hummingbot/issues/2756

### CODE REVIEW

**Comment by RC-13** [comment]
It seems that the validator only works for the second exchange/`connector_2`, any input is still accepted when setting up the market for `connector_1`:

**Review by Nullably — DISMISSED** [review]
LGTM

File: hummingbot/strategy/amm_arb/amm_arb_config_map.py [hunk 1]
```diff
@@ -59,6 +70,7 @@ def order_amount_prompt() -> str:
         key="market_1",
         prompt=market_1_prompt,
         prompt_on_new=True,
+        validator=market_1_on_validated,
```
Comment (vic-en) [inline hummingbot/strategy/amm_arb/amm_arb_config_map.py:73]:
```suggestion
        validator=market_1_validator,
```

**Comment by RC-13** [comment]
Confirmed fix, market validation now applies to both connector

**Review by RC-13 — APPROVED** [review]

**Review by Nullably — APPROVED** [review]

**Pull request merged**